packages/ui/src/working-groups/components/Worker.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { ButtonGhost } from '@/common/components/buttons'
+import { FileIcon } from '@/common/components/icons/FileIcon'
+import { MemberInfo } from '@/memberships/components'
+import { Member } from '@/memberships/types'
+
+interface WorkerProps {
+  member: Member
+  isLeader?: Member
```

Comment:
```suggestion
  isLeader?: boolean
```

It looks like we had that wrong before, could you change that?